lldb/docs/resources/lldbdap.md

```diff
@@ -1,195 +1,147 @@
+https://releases.llvm.org/ or check your systems preferred package manager for
+the `lldb` package.
 
-*Note: It's not possible to use symlinks to `lldb-dap`, as the packaging tool
-forcefully performs a deep copy of all symlinks.*
+In some cases, a language specific build of `lldb` / `lldb-dap` may also be
+available as part of the languages toolchain. For example the
+[swift language](https://www.swift.org/) toolchain includes additional language
+integrations in `lldb` and the toolchain builds provider both the `lldb` driver
+binary and `lldb-dap` binary.
 
-*Note: It's possible to use this kind flow for local installations, but it's
-not recommended because updating `lldb-dap` requires rebuilding the extension.*
+## IDE Integration
 
-## Formatting the Typescript code
+In addition to the `lldb-dap` binary, some IDEs have additional extensions to
+support debugging.
 
-This is also very simple, just run:
+- Visual Studio Code -
+[LLDB DAP Extension](https://marketplace.visualstudio.com/items?itemName=llvm-vs-code-extensions.lldb-dap)
+<!-- Add other IDE integrations. -->
 
-```bash
-npm run format
-```
+## Launching a program
 
-## Working with the VS Code extension from another extension
+To launch an executable for debugging, first define a launch configuration tells
+`lldb-dap` how to launch the binary.
 
-The VS Code extension exposes the following [VS Code
-commands](https://code.visualstudio.com/api/extension-guides/command),
-which can be invoked by other debugger extensions to leverage this extension's
-settings and logic. The commands help resolve configuration, create adapter
-descriptor, and get the lldb-dap process for state tracking, additional
-interaction, and telemetry.
+A simple launch configuration may look like
 
+```json
+{
+  "type": "lldb-dap",
+  "request": "launch",
+  "name": "Debug a.out",
+  "program": "a.out"
+}
 ```
-// Resolve debug configuration
-const resolvedConfiguration = await vscode.commands.executeCommand("lldb-dap.resolveDebugConfiguration", folder, configuration, token);
 
-// Resolve debug configuration with substituted variables
-const resolvedConfigurationWithSubstitutedVariables = await vscode.commands.executeCommand("lldb-dap.resolveDebugConfigurationWithSubstitutedVariables", folder, configuration, token);
-
-// Create debug adapter descriptor
-const adapterDescriptor = await vscode.commands.executeCommand("lldb-dap.createDebugAdapterDescriptor", session, executable);
-
-// Get DAP server process
-const process = await vscode.commands.executeCommand("lldb-dap.getServerProcess");
-```
+See the [Configuration Settings Reference](#configuration-settings-reference)
+for more information.
+
+# Supported Features
+
+`lldb-dap` supports many features of the DAP spec.
+
+- Breakpoints
+  - Source breakpoints
+  - Function breakpoint
+  - Exception breakpoints
+- Call Stacks
+- Variables
+- Watch points
+- Expression Evaluation
+- And more...
```

Comment:
I think we've reached a point where we support the majority if not almost all the features of the spec. 

I think it would be nice to have a table like https://clang.llvm.org/cxx_status.html to track our progress, but probably more something for an appendix.

Reply:
Done.